#! /usr/bin/env python

PACKAGE='realsense_wanderer'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("mEnabled", bool_t, 0, "Enabled", True)
		
gen.add("mMinY", double_t, 0, "Min Y in pointcloud", 0.0, -10.0, 10.0)
gen.add("mMaxY", double_t, 0, "Max Y in pointcloud", 2.0, -10.0, 10.0)
gen.add("mMinX", double_t, 0, "Min X in pointcloud", -0.3, -10.0, 10.0)
gen.add("mMaxX", double_t, 0, "Max X in pointcloud", 0.3, -10.0, 10.0)
gen.add("mMinZ", double_t, 0, "Min Z in pointcloud", 0.3, 0.0, 10.0)
gen.add("mMaxZ", double_t, 0, "Max Z in pointcloud", 3.0, 0.0, 10.0)

gen.add("mMinBlobSize", int_t, 0, "Min Blob size", 1000, 0, 100000)

exit(gen.generate(PACKAGE, "realsense_wanderer", "wanderer"))
